src/node_env_var.cc

```diff
@@ -379,7 +379,7 @@ static void EnvEnumerator(const PropertyCallbackInfo<Array>& info) {
 
 MaybeLocal<Object> CreateEnvVarProxy(Local<Context> context,
                                      Isolate* isolate,
-                                     Local<Object> data) {
+                                     Local<Value> data) {
```

Comment:
The last argument can be removed now since it's just empty.

Reply:
Done :+1: